--- a/src/options/stdin.rs
+++ b/src/options/stdin.rs
@@ -9,8 +9,6 @@
 use crate::options::Vars;
 use crate::options::vars::EZA_STDIN_SEPARATOR;
 use std::ffi::OsString;
-use std::io;
-use std::io::IsTerminal;
 
 #[derive(Debug, PartialEq)]
 pub enum FilesInput {
@@ -20,7 +18,7 @@
 
 impl FilesInput {
     pub fn deduce<V: Vars>(matches: &ArgMatches, vars: &V) -> Self {
-        if matches.get_flag("stdin") || !io::stdin().is_terminal() {
+        if matches.get_flag("stdin") {
             let separator = vars
                 .get(EZA_STDIN_SEPARATOR)
                 .unwrap_or(OsString::from("\n"));
