diff --git a/src/options/stdin.rs b/src/options/stdin.rs
index 7179512..d6d2962 100644
--- a/src/options/stdin.rs
+++ b/src/options/stdin.rs
@@ -8,8 +8,6 @@ use crate::options::parser::MatchedFlags;
 use crate::options::vars::EZA_STDIN_SEPARATOR;
 use crate::options::{flags, OptionsError, Vars};
 use std::ffi::OsString;
-use std::io;
-use std::io::IsTerminal;
 
 #[derive(Debug, PartialEq)]
 pub enum FilesInput {
@@ -20,7 +18,7 @@ pub enum FilesInput {
 impl FilesInput {
     pub fn deduce<V: Vars>(matches: &MatchedFlags<'_>, vars: &V) -> Result<Self, OptionsError> {
         Ok(
-            if matches.has(&flags::STDIN)? || !io::stdin().is_terminal() {
+            if matches.has(&flags::STDIN)? {
                 let separator = vars
                     .get(EZA_STDIN_SEPARATOR)
                     .unwrap_or(OsString::from("\n"));
